mlflow/server/js/src/components/modals/RenameRunFormView.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import { withRouter } from 'react-router-dom';
+import { Formik, Field } from 'formik';
+
+import { validationSchema } from './validation';
+
+/**
+ * Component that renders a form for updating a run's name. Expects to be 'closeable'
+ * (i.e. rendered within a closeable dialog) and so accepts an `onClose` callback.
+ */
+class RenameRunFormView extends Component {
+  static propTypes = {
+    onSubmit: PropTypes.func.isRequired,
+    onClose: PropTypes.func.isRequired,
+    runName: PropTypes.string.isRequired,
+    experimentId: PropTypes.number.isRequired
+  }
+
+  renderForm = (renderProps) => {
+    const {
+      handleSubmit,
+      isSubmitting,
+    } = renderProps;
+    const saveText = isSubmitting ? "Saving..." : "Save";
+    return <form onSubmit={handleSubmit} style={styles.form}>
+      <div style={styles.formField}><h2 style={{"marginTop": "0px"}}>Rename Run</h2></div>
+      <div style={styles.label}>New run name:</div>
+      <div style={{...styles.formField, "width": "100%"}}>
+        <Field
+            type="newRunName"
+            name="newRunName"
+            label="New Run Name"
+            autoFocus
+            style={{"width": "100%"}}
```

Comment:
Nit: Is there any way we could make this open with the text all selected by default, so that you can just type in a new name? I think you can add `onfocus="this.select()"` on it.